assets/src/js/vendors/lite-youtube.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable */
```

Comment:
Since we are going to maintain this code from now on, maybe it makes sense to lint it? Same for the css part.

Reply:
Sure, though maybe we can just build the script by taking it from node_modules instead :thinking: Are you ok with handling this in a follow up PR so we can take time to set it up properly and sites can already benefit from it in the meanwhile?

Reply:
We can fix the linting issues now, since it's many lines of code and probably it's mostly trivial changes, to avoid adding some technical debt here. And we iterate later on how is the best way to load the script.

Reply:
I made it use the npm package instead, so that we don't have to re-apply our code style each time we update.

Reply:
Ah much better 🙂